virt/virt-4-8-release-notes.adoc

```diff
@@ -278,3 +278,8 @@ Error creating virtual machine
 *** As a workaround, create the virtual machine from the CLI or upgrade to {VirtProductName} 4.8.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
+
+//New known issue for 4.8.2
+* If an `nmstate-handler` pod restarts, or if you apply a `NodeNetworkConfigurationPolicy` manifest to create a Linux bridge interface, the desired network policy configuration fails to be applied. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001901[*BZ#2001901*])
```

Comment:
Sorry for not making this clear for the first time, but we need to make this more specific - the issue doesn't appear when NNCP with Linux bridge is created, the issue occurrs when the policy that creates a Linux bridge is updated.

So the prerequisite for even hitting this issue is: **having a NNCP that creates a Linux bridge**
Now when there is such NNCP, the issue is hit in either:
a) a nmstate-handler pods is restarted (which re-triggers reconcile of that policy) 
b) the policy that creates that Linux bridge is updated (which again, re-triggers reconcile of that policy)

Reply:
Thanks for clarifying. It is confusing because the cause and workaround for this issue seem to be the same (re-triggering policy reconciliation).

Reply:
Yes, you are correct. It's an unpleasant bug.

Reply:
One more attempt; please review again :)